charts/consul/Chart.yaml

```diff
@@ -16,7 +16,7 @@ annotations:
   artifacthub.io/prerelease: true
   artifacthub.io/images: |
     - name: consul
-      image: hashicorp/consul:1.15.1
+      image: docker.mirror.hashicorp.services/hashicorppreview/consul-enterprise:1.16-dev
```

Comment:
This is an example of what you get when you run `make prepare-dev`. Main will be updated to this but I also needed it set to test changes properly.